apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/hooks/useEnsureActiveTabIsInTheme/useEnsureActiveTabIsInTheme.ts

```diff
@@ -0,0 +1,32 @@
+import { Blocks } from '@ballerine/blocks';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { useEffect } from 'react';
+import { toScreamingSnakeCase } from '@/common/utils/to-screaming-snake-case/to-screaming-snake-case';
+import { camelCase } from 'string-ts';
+
+export const useEnsureActiveTabIsInTheme = ({
```

Comment:
Ideally we'd build the available tabs for the URL the default and validation schema from the workflow's config.